### PR TITLE
Makefile: Add targets for Hydra OAuth2 server

### LINKED ISSUE #29: Makefile: add targets for Hydra OAuth2 server
Add targets to start, configure, stop and test Hydra OAuth2 server

### PR DESCRIPTION
**Pull request by sandrask**
Add start, stop, configure and test app targets.

Closes #29

Signed-off-by: Sandra Vrtikapa <sandra.vrtikapa@securekey.com>

### CODE REVIEW

File: test/bdd/fixtures/hydra/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#
+
+version: '3'
+
+services:
+
+  hydra:
+    image: oryd/hydra:latest
+    ports:
+      - "4444:4444" # Public port
```
Comment (rolsonquadras) [inline test/bdd/fixtures/hydra/docker-compose.yml]: Shouldn't we configure the host ports dynamically ?

Comment (sandrask) [reply test/bdd/fixtures/hydra/docker-compose.yml]: These deployment configuration improvements can be done later. I wanna first make flow work with their example defaults. I also have one to do for configuration so I can address those later on.

File: test/bdd/fixtures/hydra/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      - "4445:4445" # Admin port
+      - "5555:5555" # Port for hydra token user
+    command:
+      serve all --dangerous-force-http
+    environment:
+      - DSN=mysql://root:secret@tcp(mysqld:3306)/mysql?max_conns=20&max_idle_conns=4
```
Comment (llorllale) [inline test/bdd/fixtures/hydra/docker-compose.yml]: Anyway we can make these credentials configurable?

File: test/bdd/fixtures/hydra/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      - URLS_SELF_ISSUER=http://127.0.0.1:4444
+      - URLS_CONSENT=http://127.0.0.1:3000/consent
+      - URLS_LOGIN=http://127.0.0.1:3000/login
+      - URLS_LOGOUT=http://127.0.0.1:3000/logout
+      - DSN=memory
+      - SECRETS_SYSTEM=youReallyNeedToChangeThis
```
Comment (llorllale) [inline test/bdd/fixtures/hydra/docker-compose.yml]: You really should.

**Pull request merged**